### DISCUSSION TITLE
Frontend Dark/Light mode toggle button.

### DISCUSSION THREAD

**Discussion by ghost** [GitbookIO/community#139] [Feature Requests]
### What's your feature/idea?

Many website visitors prefer dark mode, but there are others who prefer light mode. A toggle button for switching between the two would provide visitors with the flexibility to choose which mode is best for them. The proposed feature would add a frontend dark/light theme toggle button to the GitBook interface for visitors.

Benefits:

- User-friendly interface: Users can easily switch between dark and light modes based on their preferences.
- Improved accessibility: Users who prefer dark mode can use the toggle button to make the GitBook interface more accessible to them.
- Enhanced user experience: Users can customize their GitBook interface to meet their needs, which will lead to a more enjoyable user experience overall.

### What's your desired outcome?

- The toggle button will be located in a prominent location on the GitBook interface, such as the header or footer.
- Clicking the button will switch the theme between dark and light mode.
- The selected theme will be saved for future sessions, so the user does not need to toggle it every time they open GitBook.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

### Anything else?

Thank you for considering this feature request. I believe that it would be a valuable addition to the GitBook platform and provide users with more customization options to enhance their experience.

**Comment by taszap** [comment]
Hey, @ubytedev thank you for your detailed post. 
A little update from our side: it's in the works ⏳ I can't share the exact ETA but once shipped it will be shared in our changelog and I will make sure to update you here. 

housekeeping: duplicate of #10

**Comment by taszap** [comment]
I will close this discussion in favour of the original post #10
